### ISSUE TITLE
Heracles: the conversion from int to date is unsupported

### ISSUE THREAD

**Issue by mark-velez** [OHDSI/WebAPI#70]
We are using SQL Server 2012 and the Microsoft JDBC driver. When we open the Heracles runner UI, we encounter the WebAPI exception "the conversion from int to date is unsupported." It appears that the problem occurs when executing the query `getCohortAnalysesForCohort.sql`. Specifically, it looks like the `null` value [used here](https://github.com/OHDSI/WebAPI/blob/master/src/main/resources/resources/cohortanalysis/sql/getCohortAnalysesForCohort.sql#L4) gets treated as `int` by SQL Server ([source](http://stackoverflow.com/questions/6108510/conversion-from-int-to-date-is-unsupported-on-null-date#comment-7083794)). Casting (see below) seems to solve the problem.

``` sql
SELECT a.*,
                @cohortDefinitionId AS COHORT_DEFINITION_ID,
                0 ANALYSIS_COMPLETE,
                CAST(null as datetime) as LAST_UPDATE_TIME
FROM @ohdsi_database_schema.HERACLES_ANALYSIS a
```

**Comment by evillalonNY** [comment]
I have run into this same problem with Heracles runner UI and can corroborate that  the way to solve it is with the CAST(null as datetime) as LAST_UPDATE_TIME in   file
src/main/java/resources/cohortanalysis.sql/getCohortAnalysesForCohort.sql

**Comment by anthonysena** [comment]
Closed via #86
